test/JDBC/expected/case_insensitive_collation-vu-verify.out

```diff
@@ -0,0 +1,115 @@
+select name from sys.columns where name = 'Test';
+go
+~~START~~
+varchar
+test
+~~END~~
+
+
+select name from sys.columns where name = 'TesT';
+go
+~~START~~
+varchar
+test
+~~END~~
+
+
+select COUNT(*) FROM sys.tables WHERE name = 'cameltable';
+go
+~~START~~
+int
+1
+~~END~~
+
+
+select COUNT(*) FROM sys.tables WHERE name = 'CAMELTABLE';
+go
+~~START~~
+int
+1
+~~END~~
+
+
+select COUNT(*) FROM sys.tables WHERE name = 'CaMeLtAbLe';
+go
+~~START~~
+int
+1
+~~END~~
+
+
+SELECT COUNT(*) FROM sys.columns WHERE name = 'cAmElCoLuMn';
+go
+~~START~~
+int
+1
+~~END~~
+
+
+select COUNT(*) FROM sys.views WHERE name = 'CAMELview';
+go
+~~START~~
+int
+0
+~~END~~
```

Comment:
No need to add this test for now because sys.view requires rework.

Reply:
removed